apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -18,12 +20,17 @@ export const setSelectedLessons = selected => ({
   type: SET_SELECTED_LESSONS,
   selected
 });
+export const addStudentLevelScores = scoresData => ({
+  type: ADD_STUDENT_LEVEL_SCORES,
+  scoresData
+});
```

Comment:
Please call it `setStudentLevelScores`, since it is setting them and not adding/appending to a list.